ios/Plugin/PurchasesPlugin.swift

```diff
@@ -43,6 +43,10 @@ public class PurchasesPlugin: CAPPlugin, PurchasesDelegate {
         call.resolve()
     }
 
+    @objc func setMockWebResults(_ call: CAPPluginCall) {
+        call.resolve()
```

Comment:
We are logging error on Android. Should we log in iOS too?

Reply:
Done in https://github.com/RevenueCat/purchases-capacitor/pull/97/commits/e0ac6606bedbf4b598f6842a561b908b9ada6739